roles/setup_gitops/defaults/main.yml

```
@@ -0,0 +1,6 @@
+---
+sg_default_ztp_tag: v4.19.0
+sg_default_mce_tag: v2.13
```

Do we need to make a diff between the default var and the actual var used?

By removing that difference the vars can be used directly. 

```suggestion
sg_ztp_tag: v4.19.0
sg_mce_tag: v2.13
```